invenio_github/templates/semantic-ui/invenio_github/helpers.html

```diff
@@ -6,28 +6,27 @@
   Invenio is free software; you can redistribute it and/or modify it
   under the terms of the MIT License; see LICENSE file for more details.
 #}
-{%- macro doi_badge(doi, github_id, modal_name='') %}
+{%- macro doi_badge(doi, doi_url, github_id) %}
```

Comment:
our minds aligned [here](https://github.com/inveniosoftware/invenio-app-rdm/pull/2314/files#diff-ebc60e8a8e504a64b7ee58f27622a9922b6353bc6947713bd9155eb631956e08). We should try to use the central modal when we fix the doi to concept doi